docs/raw_image.md

```diff
@@ -0,0 +1,47 @@
+
+# Instruction to boot the generated raw image to AWS
+
+Once you generate the raw image with `make raw_disk`, the generated file needs to be imported to AWS. This file describes the step to consume the image on AWS.
+
+1. Upload the image to an S3 bucket
+```
+aws s3 cp <cos-image> s3://cos-images
+```
+
+2. Created the disk container JSON (`container.json` file) as:
+
+```
+{
+  "Description": "cOS Testing image in RAW format",
+  "Format": "raw",
+  "UserBucket": {
+    "S3Bucket": "cos-images",
+    "S3Key": "<cos-image>"
+  }
+}
+```
+
+3. Import the disk as snapshot
+
+```
+aws ec2 import-snapshot --description "cOS PoC" --disk-container file://container.json
+```
+
+4. Followed the procedure described in [AWS docs](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/creating-an-ami-ebs.html#creating-launching-ami-from-snapshot) to register an AMI from snapshot. Used all default settings unless for the firmware, were I forced it to UEFI boot.
+
+5. Launch instance with this simple userdata:
```

Comment:
is this really needed? I booted the instance with a different user data, not creating users and root/cos was available.

Also, what is really needed if you want to launch the raw image is to provide a layout file (like in the example user-data) so its possible to use the new cos-deploy to install the system.

Reply:
good point! I'll update the docs :+1:

Reply:
done!